src/main/java/com/job/tracking/service/TaskService.java

```diff
@@ -0,0 +1,54 @@
+package com.job.tracking.service;
+
+import com.job.tracking.dto.UpdateTaskRequest;
+import com.job.tracking.entity.TaskEntity;
+import com.job.tracking.mapping.TaskMapper;
+import com.job.tracking.model.Task;
+import com.job.tracking.repository.TasksRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class TaskService {
```

Comment:
Please change TaskService to interface and create TaskServiceImpl with @Service annotation, it would be more readable.